modules/related-posts/jetpack-related-posts.php

```diff
@@ -76,6 +76,9 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 		if ( ! class_exists( 'Jetpack_Media_Summary' ) ) {
 			jetpack_require_lib( 'class.media-summary' );
 		}
+
```

Comment:
Thanks a lot! Somehow I thought this was already in core. I made the change you've mentioned (and will update my own plugins to fix that problem).